atd-etl/cr3_download/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# This image was used over python:3.10-slim to support python-magic library
```

Comment:
I am inexperienced on choosing base images so I went with the one that Frank used for the afd/ems one. Open to other thoughts here. I kept getting and error with `python:3.10-slim` that `libmagic` wasn't found even though it logged the pip install.

Reply:
it would be nice to use the same base image to save a little on image storage, but i think this is totally fine and definitely not worth debugging any further!

Reply:
I used 3.10 because one of the packages the cr3 extract script uses didn't list python 3.11 as compatible.